style/svgs/types/text_type.svg

```diff
@@ -1,7 +1,4 @@
-    </g>
-</svg>
\ No newline at end of file
+<svg width="14" height="10" viewBox="0 0 14 10" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
we added the dev dependency svgo (https://www.npmjs.com/package/svgo) to the main repo to optimize svg size - we could add it here too, or add in another PR